easybuild/easyconfigs/w/waLBerla/waLBerla-6.1-foss-2021a.eb

```diff
@@ -0,0 +1,34 @@
+# easyconfig file for waLBerla v6.1
+easyblock = 'CMakeMake'
+
+name = 'waLBerla'
+version = '6.1'
+
+homepage = "https://walberla.net/index.html"
+description =  '''Widely applicable Lattics-Boltzmann from Erlangen is a 
+block-structured high-performance framework for multiphysics simulations'''
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = ['https://i10git.cs.fau.de/walberla/walberla/-/archive/v6.1']
+sources = ['%(name)s-v%(version)s.tar.gz']
+checksums = ['f0acdd9ad6543bc9306c8aae953dd5065986271d4398916ae0469db8b21c007a']
+
+builddependencies = [('CMake', '3.20.1')]
+
+dependencies = [
+    ('pybind11', '2.6.2'),
+    ('Python', '3.9.5'),
+]
```

Comment:
```suggestion

]
```

```
-- Could NOT find Boost (missing: Boost_INCLUDE_DIR)
```

Reply:
Hi @branfosj, 
Thanks for the reviews. What do you mean here? Is Boost.MPI needed here as a dependency?